dimod/include/dimod/quadratic_model.h

```diff
@@ -1196,6 +1203,11 @@ class QuadraticModel : public QuadraticModelBase<Bias, Index> {
         return v;
     }
 
+    void clear() {
+        this->varinfo_.clear();
+        base_type::clear();
+    }
```

Comment:
So, `base_type::clear()` calls back to the `QuadraticModelBase` clear method above, right? I assume nothing else needs to be cleared for `BinaryQuadraticModel`, e.g., `vartype`, meaning that only the base class method is called there?

Reply:
Exactly.